lib/src/main/java/com/nextcloud/android/sso/model/FilesAppType.java

```diff
@@ -8,33 +8,19 @@
 package com.nextcloud.android.sso.model;
 
 import androidx.annotation.NonNull;
-import androidx.annotation.Nullable;
 
-public enum FilesAppType {
+public class FilesAppType {
 
-    PROD("com.nextcloud.client", "nextcloud"),
-    QA("com.nextcloud.android.qa", "nextcloud.qa"),
-    DEV("com.nextcloud.android.beta", "nextcloud.beta");
+    public static FilesAppType PROD = new FilesAppType("com.nextcloud.client", "nextcloud");
+    public static FilesAppType QA = new FilesAppType("com.nextcloud.android.qa", "nextcloud.qa");
+    public static FilesAppType DEV = new FilesAppType("com.nextcloud.android.beta", "nextcloud.beta");
```

Comment:
Please ensure, that those constants are not referenced directly within the SSO lib anywhere else. I think there might be one or two positions where we fallback to `PROD` or anything else in case we don't find the actual type. This is an mistake after this PR gets merged because it is an assumption that might be wrong when people can register or unregister types as they want.